.github/workflows/deploy.yml

```
@@ -0,0 +1,43 @@
+name: Deploy app
+
+on:
+  push:
+    branches:
+      - feat/deploy-workflow
+
+  workflow_dispatch:
+
+jobs:
+  docker_build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2.4.1
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      - name: Build the Docker image
+        run: docker build . --file Dockerfile.prod -t parnasmi/brick-breaker:latest
+
+      - name: Login hub.docker.com
+        run: docker login -u parnasmi -p ${{ secrets.DOCKER_HUB_KEY }}
+
+      - name: Push the Docker image
+        run: docker push parnasmi/brick-breaker:latest
+
+  deploy:
+    runs-on: ubuntu-latest
+    needs: [docker_build]
+
+    steps:
+      - name: Executing remote ssh commands
+        uses: appleboy/ssh-action@master
+        with:
+          host: ${{ secrets.GAME_SSH_HOST }}
+          username: ${{ secrets.GAME_SSH_NAME }}
+          key: ${{ secrets.GAME_SSH_PRIVATE_KEY }}
+          port: ${{ secrets.GAME_SSH_PORT }}
+          script: cd /web/game/ && docker pull parnasmi/brick-breaker:latest && pwd && docker-compose down && docker-compose -f docker-compose.prod.yaml up -d --build
```

Тут скорее нужно спуллить последнюю версию с гитхаба, образ с докерхаба нигде не используется в конфигах
```suggestion
          script: cd /web/game/ && git pull && docker-compose down && docker-compose -f docker-compose.prod.yaml up -d --build
```